kong/db/schema/metaschema.lua

```diff
@@ -595,6 +640,9 @@ local MetaSchema = Schema.new({
     {
       shorthand_fields = shorthand_fields_array,
     },
+    {
+      transformations = transformations_array,
+    },
```

Comment:
This move is unnecessary, strictly speaking. If possible, remove it, just to make it easier to merge on ee later. No biggie though.

Reply:
this was for style, so that we have them in both schema and subschema the same and it readable order.